slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -278,6 +278,10 @@ trait MySQLProfile extends JdbcProfile { profile =>
     override protected def dropPrimaryKey(pk: PrimaryKey): String = {
       "ALTER TABLE " + quoteTableName(table.tableNode) + " DROP PRIMARY KEY"
     }
+
+    // primary key will be dropped when the table is dropped
+    override protected def dropIfExistsPhase = Iterable(dropTable(true))
```

Comment:
As far as I know, in MySQL, indexes are deleted at the same time as the table is deleted.
I have confirmed it as following.

```sql
create table a(id int primary key);
show indexes from a;
drop table a;
create table a(id integer);
show indexes from a;
```